lib/services/ecs/index.js

```diff
@@ -45,8 +46,8 @@ function getLatestEcsAmiId() {
 
 
 function getCompiledEcsTemplate(stackName, clusterName, ownServiceContext, ownPreDeployContext, dependenciesDeployContexts, userDataScript, ecsServiceRole) {
-    return getLatestEcsAmiId()
-        .then(latestEcsAmi => {
+    return Promise.all([getLatestEcsAmiId(), route53.listHostedZones()])
+        .then(([latestEcsAmi, hostedZones]) => {
```

Comment:
Yeah, ES6/Node 6 added destructuring for arrays and objects:

`let [hi, hello] = ['ciao', 'buongiorno'];`
`let {foo, bar} = {foo: 1, bar: 2};`

I think I'm going to change this up a bit to make it more readable to someone who doesn't spot the destructuring.